candi/control-plane-kubeadm/config.yaml.tpl

```diff
@@ -3,13 +3,13 @@ RotateKubeletServerCertificate default is true, but CIS becnhmark wants it to be
 https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 */}}
 {{- $featureGates := list "TopologyAwareHints=true" "RotateKubeletServerCertificate=true" | join "," }}
-{{- if semverCompare ">= 1.26" .clusterConfiguration.kubernetesVersion }}
+{{- if semverCompare "> 1.26" .clusterConfiguration.kubernetesVersion }}
     {{- $featureGates = list $featureGates "ValidatingAdmissionPolicy=true" | join "," }}
 {{- end }}
 {{- if semverCompare "> 1.26" .clusterConfiguration.kubernetesVersion }}
     {{- $featureGates = list $featureGates "AdmissionWebhookMatchConditions=true" | join "," }}
 {{- end }}
-{{- if semverCompare "< 1.27" .clusterConfiguration.kubernetesVersion }}
+{{- if semverCompare "= 1.27" .clusterConfiguration.kubernetesVersion }}
     {{- $featureGates = list $featureGates "DaemonSetUpdateSurge=true" | join "," }}
 {{- end }}
```

Comment:
recheck the condition according to 
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/

Reply:
fix